src/api/submission/validations2/validators/fishbelt_transect.py

```diff
@@ -4,60 +4,63 @@
 from ....exceptions import check_uuid
 from ....models import FishBeltTransect
 from ....utils import get_related_transect_methods
-from .base import ERROR, OK, WARN, BaseValidator, validator_result
+from .base import ERROR, OK, ERROR, BaseValidator, validator_result
 
 
-class UniqueTransectValidator(BaseValidator):
-    INVALID_INPUTS = "invalid_transect_inputs"
-    DUPLICATE_TRANSECT = "duplicate_transect"
+class UniqueFishbeltTransectValidator(BaseValidator):
+    INVALID_DATA = "invalid_fishbelt_transect"
+    DUPLICATE_FISHBELT_TRANSECT = "duplicate_fishbelt_transect"
 
     def __init__(
         self,
         protocol,
         label_path,
         number_path,
         width_path,
-        relative_depth_path,
         site_path,
         management_path,
         sample_date_path,
         depth_path,
-        **kwargs
+        observers_path,
+        **kwargs,
     ):
         self.protocol = protocol
         self.label_path = label_path
         self.number_path = number_path
         self.width_path = width_path
-        self.relative_depth_path = relative_depth_path
         self.site_path = site_path
         self.management_path = management_path
         self.sample_date_path = sample_date_path
         self.depth_path = depth_path
+        self.observers_path = observers_path
+
         super().__init__(**kwargs)
 
     def _get_query_args(self, collect_record):
         label = self.get_value(collect_record, self.label_path) or ""
         number = self.get_value(collect_record, self.number_path) or None
         width = self.get_value(collect_record, self.width_path) or None
-        relative_depth = (
-            self.get_value(collect_record, self.relative_depth_path) or None
-        )
         site = self.get_value(collect_record, self.site_path) or None
         management = self.get_value(collect_record, self.management_path) or None
         sample_date = self.get_value(collect_record, self.sample_date_path) or None
         depth = self.get_value(collect_record, self.depth_path) or None
+        observers = self.get_value(collect_record, self.observers_path) or []
+        profiles = [o.get("profile") for o in observers]
 
         try:
             number = int(number)
             check_uuid(width)
             check_uuid(site)
             check_uuid(management)
-            if relative_depth is not None:
-                check_uuid(relative_depth)
+            float(depth)
+
             if parse_datetime(f"{sample_date} 00:00:00") is None:
```

Comment:
While we're at it, should we maybe change this to use `parse_date` without the 0s, for less chance of returning something other than `None`?